app/javascript/components/forms/login/actions.js

```diff
@@ -34,11 +34,9 @@ export const registerUser = createThunkAction('sendRegisterUser', data => () =>
 
 export const resetUserPassword = createThunkAction(
   'sendResetPassword',
-  ({ data, success }) => () =>
+  data => () =>
     resetPassword(data)
-      .then(() => {
-        success();
-      })
+      .then(() => {})
```

Comment:
do we even need this if we don't have `success`? These params come from `react-final-form` right?

Reply:
we need that `then(() => {})` as otherwise we're not rendering the confirmation message (the request is still sent though). We have a `submitSucceeded` param coming from the Form that apparently is set under the hood by `react-final-form`